### ISSUE TITLE
Unsigned exe from /Appdata blocked by AppLocker

### ISSUE THREAD

**Issue by Meowndabout** [ThePacielloGroup/CCAe#181]
Hey, hopefully you may be able to assist :) 

My team have encountered issues with the installation of this product. 

Investigations have revealed that the version of this app is not suitable within the environment as it tries to run an unsigned .exe from the users /APPDATA area which gets blocked by the Windows AppLocker. 

For this to run within our environment the user would need to be enabled to run as a local admin. This would require approval by our Security team, followed by Technology Operations which is almost impossible.

The browser version works without issue

**Comment by ferllings** [comment]
Duplicate of https://github.com/ThePacielloGroup/CCAe/issues/177

**Comment by Meowndabout** [comment]
Would the signed app try and run from Appdata still? Our Applocker policy doesn't like the folder location more than it dislikes unsigned?

**Comment by ferllings** [comment]
Difficult to say, I don't any experience with the Windows security policy.
But this issue was not mentioned in the previous (signed) versions.
I'll re-sign the app and see how it goes.

**Comment by Meowndabout** [comment]
@ferllings awesome, thank you! 😊
